### PR TITLE
Fix JSON tags on boolean Helm value struct fields (#1359)

### PR DESCRIPTION
**Pull request by marcogschmidt**
Port #1359 to `feature-rc1`.

(cherry picked from commit afad52b17def66bc629d8b3a9d9148d64018c8ae)
BOT NOTES: 
resolves https://github.com/solo-io/gloo/issues/1360

### CODE REVIEW

File: install/helm/gloo/generate/values.go [hunk 1]
```diff
@@ -229,7 +229,7 @@ type AccessLogger struct {
 	Image       *Image `json:"image,omitempty"`
 	Port        uint   `json:"port,omitempty"`
 	ServiceName string `json:"serviceName,omitempty"`
-	Enabled     bool   `json:"enabled,omitempty"`
+	Enabled     bool   `json:"enableds"`
```
Comment (ilackarms) [inline install/helm/gloo/generate/values.go]: `enableds`

Comment (ilackarms) [inline install/helm/gloo/generate/values.go]: `enableds`

Comment (kdorosh) [reply install/helm/gloo/generate/values.go]: nice catch, also an issue on the original PR: https://github.com/solo-io/gloo/pull/1359/files#diff-7261ce45321b73dc869fa8a7e8715b7dR232

let's address both

Comment (marcogschmidt) [reply install/helm/gloo/generate/values.go]: Good catch!

**Pull request merged**